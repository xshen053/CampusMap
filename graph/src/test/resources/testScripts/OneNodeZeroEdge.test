# Create a graph
CreateGraph ONZE_graph

# Add a node
AddNode ONZE_graph A

# Print the nodes in the graph and the outgoing edges from A
ListNodes ONZE_graph
ListChildren ONZE_graph A