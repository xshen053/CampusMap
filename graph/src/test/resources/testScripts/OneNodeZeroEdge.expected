# Create a graph
created graph ONZE_graph

# Add a node
added node A to ONZE_graph

# Print the nodes in the graph and the outgoing edges from A
ONZE_graph contains: A
the children of A in ONZE_graph are:

